### PR TITLE
fix: generated loader template

### LINKED ISSUE #1719: Error using a loader generated via CLI
**Describe the bug**
Error using a loader generated via CLI

**What is the current behavior?**
Syntax Error.
Screenshot at 2020-08-01 16-47-33

**To Reproduce**

1. generate loader via CLI `webpack-cli loader`
2. Go to `cd ./examples/simple`
3. Run `webpack`

**Expected behavior**
Should run without any syntax error.

**Screenshots**
See above

**Please paste the results of `webpack-cli info` here, and mention other relevant information**

**Additional context**
Fix in progress.

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
bugfix

**Did you add tests for your changes?**
Yes
**If relevant, did you update the documentation?**
NA
**Summary**
Fixes #1719 

**Does this PR introduce a breaking change?**
no

**Other information**

### CODE REVIEW

File: packages/generate-loader/templates/src/_index.js.tpl [hunk 1]
```diff
@@ -3,9 +3,9 @@
  * https://webpack.js.org/contribute/writing-a-loader
  */
 
-export default function loader(source) {
+module.exports = function loader(source) {
```
Comment (evilebottnawi) [inline packages/generate-loader/templates/src/_index.js.tpl:6]: I think we should implement `--babel` option to this in future (put babel config and generate es module syntax)

**Comment by anshumanv** [comment]
Great that we're at this,

We should also get back to https://github.com/webpack/webpack-cli/issues/1581 with proper handling, @snitin315 can you handle it next?

**Comment by snitin315** [comment]
> We should also get back to #1581 with proper handling, @snitin315 can you handle it next?

Sure 👍🏻

File: packages/generate-loader/templates/src/_index.js.tpl (hunk 1 above)
Comment (snitin315) [reply packages/generate-loader/templates/src/_index.js.tpl:6]: Will implement in future PR.

File: test/loader/loader.test.js [hunk 2]
```diff
@@ -46,5 +46,10 @@ describe('loader command', () => {
         files.forEach((file) => {
             expect(existsSync(join(__dirname, `${loaderName}/${file}`))).toBeTruthy();
         });
+
+        //check if the the generated plugin works successfully
+        const path = resolve(__dirname, './test-loader/examples/simple/');
+        stdout = run(path, [], false).stdout;
```
Comment (jamesgeorge007) [inline test/loader/loader.test.js:52]:
```suggestion
        ({ stdout } = run(path, [], false));
```
What about the destructuring approach?

Comment (snitin315) [reply test/loader/loader.test.js:52]: I think its just personal preferences, I have no problem writing it like that.

Comment (snitin315) [reply test/loader/loader.test.js:52]: Should I change this @jamesgeorge007 ?

Comment (jamesgeorge007) [reply test/loader/loader.test.js:52]: I would've went ahead with object destructuring. But yeah, as you said it's just personal preference.
/cc @webpack/cli-team

Comment (anshumanv) [reply test/loader/loader.test.js:52]: you can do it for consistency sake

**Pull request merged**